.tekton/pipeline-service-test.yaml

```diff
@@ -140,17 +148,30 @@ spec:
             - name: kubeconfig-dir
           kind: ClusterTask
           steps:
-            - name: create-podman-container
+            - name: create-ci-runner-container
               image: quay.io/redhat-pipeline-service/ci-runner:main
               resources:
                 requests:
                   memory: 500Mi
                   cpu: 300m
               script: |
                 #!/usr/bin/env bash
-                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                set -x
 
-                kubectl -n openshift-config  get secret/pull-secret -o yaml | sed 's/openshift-config/default/' | sed 's/\.dockerconfigjson/auth.json/' | grep -v type: | kubectl apply -f -
+                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+                TOKEN=$(
+                  kubectl -n openshift-config  patch secret/pull-secret -o yaml
```

Comment:
it should use `get` command to fetch the secret, the correct command is `kubectl -n openshift-config  get secret/pull-secret -o yaml`

Reply:
Fixed